share/eds/store.go

```diff
@@ -407,8 +407,9 @@ func (s *Store) getCachedAccessor(ctx context.Context, key shard.Key) (*accessor
 	if err != nil && err != errCacheMiss {
 		log.Errorf("unexpected error while reading key from bs cache %s: %s", key, err)
 	}
+	// TODO(@walldiss): need to check if err could happen with non-nil accessor
```

Comment:
The errCacheMiss can happen yet returning non-nil accessor, IIRC

Reply:
errCacheMiss means there were no accessor, so no accessor should be returned.